src/service/s**t-the-f**k-up.test.ts

```diff
@@ -0,0 +1,61 @@
+import { type Sheriff, SheriffCommand } from './s**t-the-f**k-up';
+import { type Snowflake } from '../model/id';
+import { createMockMessage } from './command-message';
+
+test('use case of stfu', async () => {
+  const executeMessage = jest.fn<Promise<void>, [Snowflake, number]>(() =>
+    Promise.resolve()
+  );
+  const sheriff: Sheriff = { executeMessage };
+  const responder = new SheriffCommand(sheriff);
+  const fn = jest.fn();
+  await responder.on(
+    'CREATE',
+    createMockMessage({
+      args: ['sftu'],
+      reply: fn
+    })
+  );
+
+  expect(fn).not.toHaveBeenCalled();
+  expect(executeMessage).not.toHaveBeenCalledWith(
```

Comment:
この引数で呼び出されていることを確認するのでこうでしょう
```suggestion
  expect(executeMessage).toHaveBeenCalledWith(
```